File: Status_22_07_2022/pattern9.py
#!/usr/bin/python3

def pattern(row):
    num,odd = 1,1
    for i in range(row):
        for j in range(i+1):
            odd +=2
            print(num,end = " ")
            num += odd
        print()
            
            
pattern(int(input("Enter the number of rows: ")))

File: Status_22_07_2022/pattern9.c
#include <stdio.h>

void pattern(int row)
{
    int num = 1,odd = 1;
    for(int i=0;i<row;i++)
    {
        for(int j=0;j<=i;j++)
        {
            printf("%d ",num);
            odd = odd+2;
            num = num + odd;
        }
        printf("\n");
    }
}

int main()
{
    int num = 0;
    printf("Enter the number of rows: ");
    scanf("%d",&num);
    pattern(num);
}